src/Manipulators/Contrast.php

```diff
@@ -34,6 +34,10 @@ public function run(Image $image)
      */
     public function getContrast()
     {
+        if (null === $this->con) {
+            return;
+        }
+
         if (!preg_match('/^-*[0-9]+$/', $this->con)) {
```

Comment:
Same change as the one I suggested for `Brightness` here too :slightly_smiling_face:.